api/internal/target/kusttarget.go

```diff
@@ -22,6 +23,7 @@ import (
 	"sigs.k8s.io/kustomize/api/resource"
 	"sigs.k8s.io/kustomize/api/types"
 	"sigs.k8s.io/kustomize/kyaml/openapi"
+
```

Comment:
Can we remove the extra newlines here and above in line 13?

Reply:
Done.